In [1]:
# This Python 3 environment comes with many helpful analytics libraries installed
# It is defined by the kaggle/python Docker image: https://github.com/kaggle/docker-python
# For example, here's several helpful packages to load

import numpy as np # linear algebra
import pandas as pd # data processing, CSV file I/O (e.g. pd.read_csv)

# Input data files are available in the read-only "../input/" directory
# For example, running this (by clicking run or pressing Shift+Enter) will list all files under the input directory

import os
for dirname, _, filenames in os.walk('/kaggle/input'):
    for filename in filenames:
        print(os.path.join(dirname, filename))

# You can write up to 20GB to the current directory (/kaggle/working/) that gets preserved as output when you create a version using "Save & Run All" 
# You can also write temporary files to /kaggle/temp/, but they won't be saved outside of the current session

/kaggle/input/supermarket-sales/supermarket_sales - Sheet1.csv


In [2]:
# Importing the data set

df = pd.read_csv('/kaggle/input/supermarket-sales/supermarket_sales - Sheet1.csv')

In [3]:
# Setting to display all the columns

pd.get_option("display.max_columns")

20

In [4]:
# Display first five rows

df.head()

Invoice ID Branch       City Customer type  Gender  \
0  750-67-8428      A     Yangon        Member  Female   
1  226-31-3081      C  Naypyitaw        Normal  Female   
2  631-41-3108      A     Yangon        Normal    Male   
3  123-19-1176      A     Yangon        Member    Male   
4  373-73-7910      A     Yangon        Normal    Male   

             Product line  Unit price  Quantity   Tax 5%     Total       Date  \
0       Health and beauty       74.69         7  26.1415  548.9715   1/5/2019   
1  Electronic accessories       15.28         5   3.8200   80.2200   3/8/2019   
2      Home and lifestyle       46.33         7  16.2155  340.5255   3/3/2019   
3       Health and beauty       58.22         8  23.2880  489.0480  1/27/2019   
4       Sports and travel       86.31         7  30.2085  634.3785   2/8/2019   

    Time      Payment    cogs  gross margin percentage  gross income  Rating  
0  13:08      Ewallet  522.83                 4.761905       26.1415     9.1  
1  10:29         Cash   76.40                 4.761905        3.8200     9.6  
2  13:23  Credit card  324.31                 4.761905       16.2155     7.4  
3  20:33      Ewallet  465.76                 4.761905       23.2880     8.4  
4  10:37      Ewallet  604.17                 4.761905       30.2085     5.3

In [5]:
# Checking for missing Values.
df.isnull().sum()

Invoice ID                 0
Branch                     0
City                       0
Customer type              0
Gender                     0
Product line               0
Unit price                 0
Quantity                   0
Tax 5%                     0
Total                      0
Date                       0
Time                       0
Payment                    0
cogs                       0
gross margin percentage    0
gross income               0
Rating                     0
dtype: int64

In [6]:
df.describe()

Unit price     Quantity       Tax 5%        Total        cogs  \
count  1000.000000  1000.000000  1000.000000  1000.000000  1000.00000   
mean     55.672130     5.510000    15.379369   322.966749   307.58738   
std      26.494628     2.923431    11.708825   245.885335   234.17651   
min      10.080000     1.000000     0.508500    10.678500    10.17000   
25%      32.875000     3.000000     5.924875   124.422375   118.49750   
50%      55.230000     5.000000    12.088000   253.848000   241.76000   
75%      77.935000     8.000000    22.445250   471.350250   448.90500   
max      99.960000    10.000000    49.650000  1042.650000   993.00000   

       gross margin percentage  gross income      Rating  
count              1000.000000   1000.000000  1000.00000  
mean                  4.761905     15.379369     6.97270  
std                   0.000000     11.708825     1.71858  
min                   4.761905      0.508500     4.00000  
25%                   4.761905      5.924875     5.50000  
50%                   4.761905     12.088000     7.00000  
75%                   4.761905     22.445250     8.50000  
max                   4.761905     49.650000    10.00000

It is clear that there is no missing value in the dataframe


In [7]:
# displaying the datatypes of each columns

df.dtypes

Invoice ID                  object
Branch                      object
City                        object
Customer type               object
Gender                      object
Product line                object
Unit price                 float64
Quantity                     int64
Tax 5%                     float64
Total                      float64
Date                        object
Time                        object
Payment                     object
cogs                       float64
gross margin percentage    float64
gross income               float64
Rating                     float64
dtype: object

There are combination of "Object" and "float"

In [8]:
int_c = obj_c = 0
for i in df.dtypes:
    if i == 'object':
        obj_c +=1
    else:
        int_c +=1

In [9]:
print("Total columns with Numerical data type is ",int_c)
print("Total columns with object data type is ",obj_c)

Total columns with Numerical data type is  8
Total columns with object data type is  9


In [10]:
# Counting the total number of branches

branch = df.groupby(["Branch"])["Branch"].count().sort_values(ascending=False)
branch

Branch
A    340
B    332
C    328
Name: Branch, dtype: int64

##### **Among the Three Branches Sales achieved by Branch "A" is higher which is around 340, and sales of branch "B" and branch "c" is 332 and 328**

In [11]:
branch.to_csv("Branch.csv")

In [12]:
#counting the total number of cities
city = df.groupby(["City"])['City'].count().sort_values(ascending=False)
city
city.to_csv("city.csv")

In [13]:
city

City
Yangon       340
Mandalay     332
Naypyitaw    328
Name: City, dtype: int64

#### **Branch A is in city "Yangon", Branch B is in city "Mandalay", Branch C is in city "Naypyitaw"**

## Customer types

In [14]:
df['Customer type'].unique()

array(['Member', 'Normal'], dtype=object)

There are Two types of customer 
1. Member
2. Normal

In [15]:
customer_type = df.groupby(["Customer type"])['Customer type'].count().sort_values(ascending=False)
customer_type.to_csv("customer_type.csv")
customer_type

Customer type
Member    501
Normal    499
Name: Customer type, dtype: int64

In [16]:
import os
for dirname, _, filenames in os.walk('/kaggle'):
    for filename in filenames:
        print(os.path.join(dirname, filename))
cdf = pd.read_csv('//kaggle/working/customer_type.csv')

/kaggle/src/script.ipynb
/kaggle/lib/kaggle/gcp.py
/kaggle/input/supermarket-sales/supermarket_sales - Sheet1.csv
/kaggle/working/Branch.csv
/kaggle/working/customer_type.csv
/kaggle/working/city.csv
/kaggle/working/__notebook__.ipynb


In [17]:
cdf.head()

Customer type  Customer type.1
0        Member              501
1        Normal              499

In [18]:
cdf.columns=['Customer type','count']

In [19]:
cdf['Avg'] = (cdf['count'] / cdf['count'].sum()) * 100
cdf.to_csv("customer_type.csv",index=False)

## Gender

In [20]:
df['Gender'].unique()

array(['Female', 'Male'], dtype=object)

In [21]:
gender = df['Gender'].value_counts()
gender.to_csv("gender.csv")

In [22]:
gdf = pd.read_csv("//kaggle/working/gender.csv")
gdf.columns=['Gender','Count']
gdf['Avg'] = (gdf['Count']/gdf['Count'].sum())*100
gdf.to_csv("gender.csv",index=False)
gdf

Gender  Count   Avg
0  Female    501  50.1
1    Male    499  49.9

## Product line

In [23]:
df.head()

Invoice ID Branch       City Customer type  Gender  \
0  750-67-8428      A     Yangon        Member  Female   
1  226-31-3081      C  Naypyitaw        Normal  Female   
2  631-41-3108      A     Yangon        Normal    Male   
3  123-19-1176      A     Yangon        Member    Male   
4  373-73-7910      A     Yangon        Normal    Male   

             Product line  Unit price  Quantity   Tax 5%     Total       Date  \
0       Health and beauty       74.69         7  26.1415  548.9715   1/5/2019   
1  Electronic accessories       15.28         5   3.8200   80.2200   3/8/2019   
2      Home and lifestyle       46.33         7  16.2155  340.5255   3/3/2019   
3       Health and beauty       58.22         8  23.2880  489.0480  1/27/2019   
4       Sports and travel       86.31         7  30.2085  634.3785   2/8/2019   

    Time      Payment    cogs  gross margin percentage  gross income  Rating  
0  13:08      Ewallet  522.83                 4.761905       26.1415     9.1  
1  10:29         Cash   76.40                 4.761905        3.8200     9.6  
2  13:23  Credit card  324.31                 4.761905       16.2155     7.4  
3  20:33      Ewallet  465.76                 4.761905       23.2880     8.4  
4  10:37      Ewallet  604.17                 4.761905       30.2085     5.3

In [24]:
grp=df.groupby(['Product line']).mean()
grp

Unit price  Quantity     Tax 5%       Total  \
Product line                                                          
Electronic accessories   53.551588  5.711765  15.220597  319.632538   
Fashion accessories      57.153652  5.067416  14.528062  305.089298   
Food and beverages       56.008851  5.471264  15.365310  322.671517   
Health and beauty        54.854474  5.618421  15.411572  323.643020   
Home and lifestyle       55.316937  5.693750  16.030331  336.636956   
Sports and travel        56.993253  5.542169  15.812630  332.065220   

                              cogs  gross margin percentage  gross income  \
Product line                                                                
Electronic accessories  304.411941                 4.761905     15.220597   
Fashion accessories     290.561236                 4.761905     14.528062   
Food and beverages      307.306207                 4.761905     15.365310   
Health and beauty       308.231447                 4.761905     15.411572   
Home and lifestyle      320.606625                 4.761905     16.030331   
Sports and travel       316.252590                 4.761905     15.812630   

                          Rating  
Product line                      
Electronic accessories  6.924706  
Fashion accessories     7.029213  
Food and beverages      7.113218  
Health and beauty       7.003289  
Home and lifestyle      6.837500  
Sports and travel       6.916265

In [25]:
grp[['Unit price']].to_csv("product inline.csv")
grp

Unit price  Quantity     Tax 5%       Total  \
Product line                                                          
Electronic accessories   53.551588  5.711765  15.220597  319.632538   
Fashion accessories      57.153652  5.067416  14.528062  305.089298   
Food and beverages       56.008851  5.471264  15.365310  322.671517   
Health and beauty        54.854474  5.618421  15.411572  323.643020   
Home and lifestyle       55.316937  5.693750  16.030331  336.636956   
Sports and travel        56.993253  5.542169  15.812630  332.065220   

                              cogs  gross margin percentage  gross income  \
Product line                                                                
Electronic accessories  304.411941                 4.761905     15.220597   
Fashion accessories     290.561236                 4.761905     14.528062   
Food and beverages      307.306207                 4.761905     15.365310   
Health and beauty       308.231447                 4.761905     15.411572   
Home and lifestyle      320.606625                 4.761905     16.030331   
Sports and travel       316.252590                 4.761905     15.812630   

                          Rating  
Product line                      
Electronic accessories  6.924706  
Fashion accessories     7.029213  
Food and beverages      7.113218  
Health and beauty       7.003289  
Home and lifestyle      6.837500  
Sports and travel       6.916265

## product in each category

In [26]:
grp = df.groupby(['Product line']).count()
grp

Invoice ID  Branch  City  Customer type  Gender  \
Product line                                                              
Electronic accessories         170     170   170            170     170   
Fashion accessories            178     178   178            178     178   
Food and beverages             174     174   174            174     174   
Health and beauty              152     152   152            152     152   
Home and lifestyle             160     160   160            160     160   
Sports and travel              166     166   166            166     166   

                        Unit price  Quantity  Tax 5%  Total  Date  Time  \
Product line                                                              
Electronic accessories         170       170     170    170   170   170   
Fashion accessories            178       178     178    178   178   178   
Food and beverages             174       174     174    174   174   174   
Health and beauty              152       152     152    152   152   152   
Home and lifestyle             160       160     160    160   160   160   
Sports and travel              166       166     166    166   166   166   

                        Payment  cogs  gross margin percentage  gross income  \
Product line                                                                   
Electronic accessories      170   170                      170           170   
Fashion accessories         178   178                      178           178   
Food and beverages          174   174                      174           174   
Health and beauty           152   152                      152           152   
Home and lifestyle          160   160                      160           160   
Sports and travel           166   166                      166           166   

                        Rating  
Product line                    
Electronic accessories     170  
Fashion accessories        178  
Food and beverages         174  
Health and beauty          152  
Home and lifestyle         160  
Sports and travel          166

In [27]:
grp['Branch'].to_csv("Count of product.csv")
cop = pd.read_csv("//kaggle/working/Count of product.csv")
cop.columns=['Product line','count']
cop

Product line  count
0  Electronic accessories    170
1     Fashion accessories    178
2      Food and beverages    174
3       Health and beauty    152
4      Home and lifestyle    160
5       Sports and travel    166

In [28]:
grp1 = df.groupby("Product line").mean()

for i in range(0,len(cop)):
    cop['Avg price'][i]= (grp1['Unit price'][i])
cop.to_csv("product inline.csv",index=False)
cop

KeyError: 'Avg price'

## To find sum of  amount in each product inline


In [29]:
df.head()

Invoice ID Branch       City Customer type  Gender  \
0  750-67-8428      A     Yangon        Member  Female   
1  226-31-3081      C  Naypyitaw        Normal  Female   
2  631-41-3108      A     Yangon        Normal    Male   
3  123-19-1176      A     Yangon        Member    Male   
4  373-73-7910      A     Yangon        Normal    Male   

             Product line  Unit price  Quantity   Tax 5%     Total       Date  \
0       Health and beauty       74.69         7  26.1415  548.9715   1/5/2019   
1  Electronic accessories       15.28         5   3.8200   80.2200   3/8/2019   
2      Home and lifestyle       46.33         7  16.2155  340.5255   3/3/2019   
3       Health and beauty       58.22         8  23.2880  489.0480  1/27/2019   
4       Sports and travel       86.31         7  30.2085  634.3785   2/8/2019   

    Time      Payment    cogs  gross margin percentage  gross income  Rating  
0  13:08      Ewallet  522.83                 4.761905       26.1415     9.1  
1  10:29         Cash   76.40                 4.761905        3.8200     9.6  
2  13:23  Credit card  324.31                 4.761905       16.2155     7.4  
3  20:33      Ewallet  465.76                 4.761905       23.2880     8.4  
4  10:37      Ewallet  604.17                 4.761905       30.2085     5.3

In [30]:
grp2=df.groupby('Product line').sum()
grp2['Total']

Product line
Electronic accessories    54337.5315
Fashion accessories       54305.8950
Food and beverages        56144.8440
Health and beauty         49193.7390
Home and lifestyle        53861.9130
Sports and travel         55122.8265
Name: Total, dtype: float64

In [31]:
cop

Product line  count
0  Electronic accessories    170
1     Fashion accessories    178
2      Food and beverages    174
3       Health and beauty    152
4      Home and lifestyle    160
5       Sports and travel    166

In [32]:
cop['Total amount'] = grp2['Total']
for i in range(0,len(cop)):
    cop['Total amount'][i]= (grp2['Total'][i])
cop.to_csv("product inline.csv",index=False)
cop

/opt/conda/lib/python3.7/site-packages/ipykernel_launcher.py:3: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  This is separate from the ipykernel package so we can avoid doing imports until


Product line  count  Total amount
0  Electronic accessories    170    54337.5315
1     Fashion accessories    178    54305.8950
2      Food and beverages    174    56144.8440
3       Health and beauty    152    49193.7390
4      Home and lifestyle    160    53861.9130
5       Sports and travel    166    55122.8265

In [33]:
grp2

Unit price  Quantity     Tax 5%       Total      cogs  \
Product line                                                                    
Electronic accessories     9103.77       971  2587.5015  54337.5315  51750.03   
Fashion accessories       10173.35       902  2585.9950  54305.8950  51719.90   
Food and beverages         9745.54       952  2673.5640  56144.8440  53471.28   
Health and beauty          8337.88       854  2342.5590  49193.7390  46851.18   
Home and lifestyle         8850.71       911  2564.8530  53861.9130  51297.06   
Sports and travel          9460.88       920  2624.8965  55122.8265  52497.93   

                        gross margin percentage  gross income  Rating  
Product line                                                           
Electronic accessories               809.523810     2587.5015  1177.2  
Fashion accessories                  847.619048     2585.9950  1251.2  
Food and beverages                   828.571429     2673.5640  1237.7  
Health and beauty                    723.809524     2342.5590  1064.5  
Home and lifestyle                   761.904762     2564.8530  1094.0  
Sports and travel                    790.476190     2624.8965  1148.1

In [34]:
cop['Tax'] = grp2['Tax 5%']
for i in range(0,len(cop)):
    cop['Tax'][i]= (grp2['Tax 5%'][i])
cop.to_csv("product inline.csv",index=False)
cop

/opt/conda/lib/python3.7/site-packages/ipykernel_launcher.py:3: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  This is separate from the ipykernel package so we can avoid doing imports until


Product line  count  Total amount        Tax
0  Electronic accessories    170    54337.5315  2587.5015
1     Fashion accessories    178    54305.8950  2585.9950
2      Food and beverages    174    56144.8440  2673.5640
3       Health and beauty    152    49193.7390  2342.5590
4      Home and lifestyle    160    53861.9130  2564.8530
5       Sports and travel    166    55122.8265  2624.8965

In [35]:
df.head()

Invoice ID Branch       City Customer type  Gender  \
0  750-67-8428      A     Yangon        Member  Female   
1  226-31-3081      C  Naypyitaw        Normal  Female   
2  631-41-3108      A     Yangon        Normal    Male   
3  123-19-1176      A     Yangon        Member    Male   
4  373-73-7910      A     Yangon        Normal    Male   

             Product line  Unit price  Quantity   Tax 5%     Total       Date  \
0       Health and beauty       74.69         7  26.1415  548.9715   1/5/2019   
1  Electronic accessories       15.28         5   3.8200   80.2200   3/8/2019   
2      Home and lifestyle       46.33         7  16.2155  340.5255   3/3/2019   
3       Health and beauty       58.22         8  23.2880  489.0480  1/27/2019   
4       Sports and travel       86.31         7  30.2085  634.3785   2/8/2019   

    Time      Payment    cogs  gross margin percentage  gross income  Rating  
0  13:08      Ewallet  522.83                 4.761905       26.1415     9.1  
1  10:29         Cash   76.40                 4.761905        3.8200     9.6  
2  13:23  Credit card  324.31                 4.761905       16.2155     7.4  
3  20:33      Ewallet  465.76                 4.761905       23.2880     8.4  
4  10:37      Ewallet  604.17                 4.761905       30.2085     5.3

## date

In [36]:
grp3=df.groupby('Date').count()
grp3['Invoice ID'].to_csv("Date.csv")


In [37]:
date_df= pd.read_csv("//kaggle/working/Date.csv")
date_df.columns = ['Date','Number of purchases']
date_df.to_csv("Date.csv",index=False)

## Mode of purchases

In [38]:
grp4 = df.groupby('Payment').count()
grp4['Branch'].to_csv("Mode of purchase.csv")

In [39]:
mod = pd.read_csv("//kaggle/working/Mode of purchase.csv")
mod.columns=['Mode','Count']
mod

Mode  Count
0         Cash    344
1  Credit card    311
2      Ewallet    345

In [40]:
mod['percent'] = (mod['Count']/mod['Count'].sum()) * 100
mod.to_csv("Mode of purchase.csv",index=False)
mod

Mode  Count  percent
0         Cash    344     34.4
1  Credit card    311     31.1
2      Ewallet    345     34.5

In [41]:
df

Invoice ID Branch       City Customer type  Gender  \
0    750-67-8428      A     Yangon        Member  Female   
1    226-31-3081      C  Naypyitaw        Normal  Female   
2    631-41-3108      A     Yangon        Normal    Male   
3    123-19-1176      A     Yangon        Member    Male   
4    373-73-7910      A     Yangon        Normal    Male   
..           ...    ...        ...           ...     ...   
995  233-67-5758      C  Naypyitaw        Normal    Male   
996  303-96-2227      B   Mandalay        Normal  Female   
997  727-02-1313      A     Yangon        Member    Male   
998  347-56-2442      A     Yangon        Normal    Male   
999  849-09-3807      A     Yangon        Member  Female   

               Product line  Unit price  Quantity   Tax 5%      Total  \
0         Health and beauty       74.69         7  26.1415   548.9715   
1    Electronic accessories       15.28         5   3.8200    80.2200   
2        Home and lifestyle       46.33         7  16.2155   340.5255   
3         Health and beauty       58.22         8  23.2880   489.0480   
4         Sports and travel       86.31         7  30.2085   634.3785   
..                      ...         ...       ...      ...        ...   
995       Health and beauty       40.35         1   2.0175    42.3675   
996      Home and lifestyle       97.38        10  48.6900  1022.4900   
997      Food and beverages       31.84         1   1.5920    33.4320   
998      Home and lifestyle       65.82         1   3.2910    69.1110   
999     Fashion accessories       88.34         7  30.9190   649.2990   

          Date   Time      Payment    cogs  gross margin percentage  \
0     1/5/2019  13:08      Ewallet  522.83                 4.761905   
1     3/8/2019  10:29         Cash   76.40                 4.761905   
2     3/3/2019  13:23  Credit card  324.31                 4.761905   
3    1/27/2019  20:33      Ewallet  465.76                 4.761905   
4     2/8/2019  10:37      Ewallet  604.17                 4.761905   
..         ...    ...          ...     ...                      ...   
995  1/29/2019  13:46      Ewallet   40.35                 4.761905   
996   3/2/2019  17:16      Ewallet  973.80                 4.761905   
997   2/9/2019  13:22         Cash   31.84                 4.761905   
998  2/22/2019  15:33         Cash   65.82                 4.761905   
999  2/18/2019  13:28         Cash  618.38                 4.761905   

     gross income  Rating  
0         26.1415     9.1  
1          3.8200     9.6  
2         16.2155     7.4  
3         23.2880     8.4  
4         30.2085     5.3  
..            ...     ...  
995        2.0175     6.2  
996       48.6900     4.4  
997        1.5920     7.7  
998        3.2910     4.1  
999       30.9190     6.6  

[1000 rows x 17 columns]

## Rating

In [42]:
grp5=df.groupby("Rating").max().sort_values(ascending=False,by='Rating')
grp5.to_csv("Rating.csv")
grp5

Invoice ID Branch    City Customer type Gender       Product line  \
Rating                                                                       
10.0    866-70-2814      C  Yangon        Normal   Male  Sports and travel   
9.9     868-52-7573      C  Yangon        Normal   Male  Sports and travel   
9.8     805-86-0265      C  Yangon        Normal   Male  Sports and travel   
9.7     746-68-6593      C  Yangon        Normal   Male  Sports and travel   
9.6     875-31-8302      C  Yangon        Normal   Male  Sports and travel   
...             ...    ...     ...           ...    ...                ...   
4.4     870-76-1733      C  Yangon        Normal   Male  Sports and travel   
4.3     826-58-8051      C  Yangon        Normal   Male  Sports and travel   
4.2     895-03-6665      C  Yangon        Normal   Male  Sports and travel   
4.1     871-39-9221      C  Yangon        Normal   Male  Sports and travel   
4.0     885-56-0389      C  Yangon        Normal   Male  Sports and travel   

        Unit price  Quantity   Tax 5%      Total       Date   Time  Payment  \
Rating                                                                        
10.0         93.39        10  28.0170   588.3570  3/27/2019  19:18  Ewallet   
9.9          99.69        10  43.5500   914.5500  3/28/2019  19:40  Ewallet   
9.8          93.96         9  42.2820   887.9220   3/9/2019  20:54  Ewallet   
9.7          87.16        10  29.9925   629.8425   3/7/2019  19:17  Ewallet   
9.6          95.54        10  44.7400   939.5400   3/9/2019  20:29  Ewallet   
...            ...       ...      ...        ...        ...    ...      ...   
4.4          97.38        10  48.6900  1022.4900   3/7/2019  20:51  Ewallet   
4.3          88.43        10  41.1700   864.5700   3/3/2019  20:47  Ewallet   
4.2          99.96        10  44.9820   944.6220   3/9/2019  20:21  Ewallet   
4.1          93.14         9  35.5160   745.8360  3/26/2019  20:38  Ewallet   
4.0          98.53         9  32.7960   688.7160   3/5/2019  20:06  Ewallet   

          cogs  gross margin percentage  gross income  
Rating                                                 
10.0    560.34                 4.761905       28.0170  
9.9     871.00                 4.761905       43.5500  
9.8     845.64                 4.761905       42.2820  
9.7     599.85                 4.761905       29.9925  
9.6     894.80                 4.761905       44.7400  
...        ...                      ...           ...  
4.4     973.80                 4.761905       48.6900  
4.3     823.40                 4.761905       41.1700  
4.2     899.64                 4.761905       44.9820  
4.1     710.32                 4.761905       35.5160  
4.0     655.92                 4.761905       32.7960  

[61 rows x 16 columns]